app/controllers/application_controller.rb

```diff
@@ -61,19 +54,23 @@ def authorize_resource(resource, action)
   def check_no_institution!
     return if current_user && current_user.need_change_password?
     if current_user && current_user.institutions.empty? && current_user.policies.empty?
-      redirect_to new_institution_path
+      if has_access?(Institution, CREATE_INSTITUTION)
+        redirect_to new_institution_path
+      else
+        @hide_nav_bar = true
```

Comment:
Eventually, when the user has the ability to request access to an institution he will be redirected to that page.